scipy/sparse/_base.py

```diff
@@ -310,9 +310,54 @@ def format(self) -> str:
         """Format string for matrix."""
         return self._format
 
+    @property
+    def A(self) -> np.ndarray:
+        """DEPRECATED: Return a dense array.
```

Comment:
Side-note: Should we remove the usage of those deprecated properties in SciPy then?

Reply:
Is removing usage of deprecated properties from usage in SciPy something we should do in this PR or can/should it be a separate PR?